File: src/main/java/com/distribution/modules/api/pojo/vo/POSVO.java
package com.distribution.modules.api.pojo.vo;

import lombok.Data;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 * @Description TODO(描述)
 * @create 2018/6/20-21:41
 */
@Data
public class POSVO {
    /**
     *
     */
    private String name;
    /**
     *
     */
    private Long mobile;
    /**
     * 商户名称
     */
    private String merchants;
    /**
     * 区域编码
     */
    private String areaCode;
    /**
     * 连锁属性 0:非连锁,1:连锁
     */
    private Integer attribute;
}

File: src/main/kotlin/com/distribution/modules/api/pojo/vo/POSVO.kt
package com.distribution.modules.api.pojo.vo

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 
 * @create 2018/6/20-21:41
 */

data class POSVO(
        /**
         *
         */
        var name: String = "",
        /**
         *
         */
        var mobile: Long = 0L,
        /**
         * 商户名称
         */
        var merchants: String = "",
        /**
         * 区域编码
         */
        var areaCode: String = "",
        /**
         * 连锁属性 0:非连锁,1:连锁
         */
        var attribute: Int = 0

)
